components/typography/styles.js

```diff
@@ -330,53 +330,51 @@ const fonts = {
 	BCSansItalic: 'BCSans-Italic',
 	BCSansBoldItalic: 'BCSans-BoldItalic'
 };
-export const fontFacesCss = `
-	@font-face {
-		font-family: 'Lato';
-		font-style: normal;
-		font-weight: 400;
-		src: local('Lato Regular'), local('Lato-Regular'), url(${new URL(`${fonts.LatoRegular}.woff2`, importUrl)}) format('woff2'), url(${new URL(`${fonts.LatoRegular}.woff`, importUrl)}) format('woff'), url(${new URL(`${fonts.LatoRegular}.ttf`, importUrl)}) format('truetype');
-	}
-	@font-face {
-		font-family: 'Lato';
-		font-style: normal;
-		font-weight: 700;
-		src: local('Lato Bold'), local('Lato-Bold'), url(${new URL(`${fonts.LatoBold}.woff2`, importUrl)}) format('woff2'), url(${new URL(`${fonts.LatoBold}.woff`, importUrl)}) format('woff'), url(${new URL(`${fonts.LatoBold}.ttf`, importUrl)}) format('truetype');
-	}
```

Comment:
There are only 2 changes here, but because I un-indented it by one it looks worse. The Sass wasn't including the `local('Lato Regular')` (and bold), but this was. Also the Sass wasn't including the TTF assets, which shouldn't have been required [since IE8](https://caniuse.com/woff). I went with the Sass as the source of truth since that's what's been getting included in every Brightspace page.

Reply:
This change actually broke all the vdiffs that were explicitly importing `typography.js`. I reverted it to confirm this, then restore it. That means the runners have a local copy of Lato that's different from the one we have on the CDN! So I'm good treating this change of removing them as "correct".